### ISSUE TITLE
Improve the API for mentions feature

### ISSUE THREAD

**Issue by itaditya** [dear-github/dear-github#294]
There is no solid support for mentions in the Github Api. I request to add two features.

1. Mentions Webhook Event - Whenever someone mentions me fire a webhook.
2. In the Comments Api show a list of mentions for each comment

**Comment by rbipin** [comment]
yes, this webhook would be very useful. A webhook may be at the user account level, that can push notifications when the user is mentioned.

**Comment by ori78** [comment]
Agreed! Any roadmap for this?
